ADDfitUtils.py

```diff
@@ -0,0 +1,287 @@
+def doFitOnGraph(params, lvals, xvals, xerrs,
```

Comment:
Is it necessary to duplicate all this code and all the files related to the fit? I would prefer if you could find a why to integrate the changes necessary for the ADD and make it configurable. Less code to maintain is always good. :)